woocommerce/woocommerce
# Update the recommended engine versions to use

Just updating the node/pnpm versions to use replacing the npm as we no longer use that.

Testing:
* Run `nvm use`.
* Run `pnpm install` and make sure everything is installed without issues.
